synthetic_data/distinct_generators/text_generator.py

```diff
@@ -39,6 +39,10 @@ def random_text(
         )
     text_list = []
 
+    # edge case
+    if str_len_min == str_len_max:
+        str_len_max += 1
+
```

Comment:
Another edge case where str_len_max == str_len_min in report. Doesn't work in rng.integers, so add one to to str_len_max as it's non-inclusive in function